azext_iot/tests/iothub/configurations/test_iot_config_int.py

```diff
@@ -96,6 +96,9 @@ def test_edge_deployments(self):
                 edge_content_malformed_path
             )
             self.kwargs["labels"] = '{"key0": "value0"}'
+            self.kwargs["custom_labels"] = "key0=value0"
+            self.kwargs["custom_metric_queries"] = "mymetric=SELECT deviceId FROM devices {}".format(
```

Comment:
Can we please use a different test input value compared to existing inline json input tests so we can be sure kvp input transformation works?